python-avd/pyavd/_eos_cli_config_gen/schema/schema_fragments/ethernet_interfaces.schema.yml

```diff
@@ -1945,6 +1945,200 @@ keys:
+                Tap ports receive traffic for replication on one or more tool ports.
+              type: dict
+              keys:
+                allowed_vlan:
+                  type: str
+                  convert_types:
+                    - int
+                  description: VLAN ID or range(s) of VLAN IDs within range 1-4094.
+                default:
+                  description: Default tap destination config.
+                  type: dict
+                  keys:
+                    groups:
+                      description: Tap group names for the interface.
+                      type: list
+                      items:
+                        type: str
+                    interfaces:
+                      description: Interfaces like -  Ethernet, InternalRecirc, Port-Channel, Recirc-Channel.
+                      type: list
+                      items:
+                        type: str
+                    nexthop_groups:
+                      type: list
+                      description: Default nexthop-group names.
+                      items:
+                        type: str
+                identity:
+                  type: dict
+                  keys:
+                    id:
+                      type: int
+                      convert_types:
+                        - str
+                      min: 1
+                      max: 65535
+                      description: Tap port VLAN ID (1-4094) or DzGRE extended ID (1-65535).
+                    inner_vlan:
+                      description: Tap port inner VLAN ID. Only applicable if `id` is a VLAN ID (1-4094).
+                      type: int
+                      convert_types:
+                        - str
+                      min: 1
+                      max: 4094
+                mpls_pop_all:
+                  type: bool
+                  description: Pop all MPLS labels.
+                native_vlan:
+                  type: int
+                  convert_types:
+                    - str
+                  min: 1
+                  max: 4094
+                  description: Native VLAN ID when interface is in tap mode.
+                truncation:
+                  type: dict
+                  keys:
+                    enabled:
+                      type: bool
+                      required: true
+                    size:
+                      description: Ingress packet truncation size in bytes.
+                      type: int
+                      convert_types:
+                        - str
+                      min: 100
+                      max: 9236
+                mac_address:
+                  type: dict
+                  keys:
+                    src:
+                      type: str
+                      pattern: "^([0-9a-f]{2}:){5}[0-9a-f]{2}$"
+                      description: MAC address for the source.
+                    dest:
+                      type: str
+                      required: true
```

Comment:
please avoid the required here if we can just check for it in the template.

Reply:
Fixed!